style.css

```diff
@@ -113,7 +138,9 @@ Wrappers (container, flexbox, etc.)
 /* Container */
 
 /* Wrappers */
-
+.wrapper {
+	max-width: 85%;
```

Comment:
We should use a pixel value here, whatever the width is in Figma. We can still use the % for `width`, but `max-width` should be pixels so that is does not get too big on super large screens (like your TV 😉 ).

Reply:
Changed to max width of 1728px, do I still include "width: 85%"? (that % value may change, it was just a placeholder)

Reply:
Yes, keep that in there for the smaller screen sizes.